### PR TITLE
Warnings displayed by default

### PR DESCRIPTION
**Pull request by nicoddemus**
Merge of #1704 into up to date `features` (supersedes it).

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -160,6 +160,10 @@ time or change existing behaviors in order to make them less surprising/more use
   parametrize.
   Thanks `@palaviv`_ for the complete PR (`#1474`_).
 
+* Now pytest warnings summary is shown up by default. Added a new flag
+  ``--disable-pytest-warnings`` to explicitly disable the warnings summary.
+  This change resolves the (`#1668`_).
```
Comment (The-Compiler) [inline CHANGELOG.rst]: Aaand we have weird changelog entries again :wink:

File: CHANGELOG.rst [hunk 2]
```diff
@@ -216,6 +220,8 @@ time or change existing behaviors in order to make them less surprising/more use
 * Fix internal error issue when the ``method`` argument is missing for
   ``teardown_method()`` (`#1605`_).
 
+* Renamed the pytest ``pdb`` module (plugin) into ``debugging``.
+
```
Comment (The-Compiler) [inline CHANGELOG.rst]: This surely shouldn't be added as part of this PR?

File: CHANGELOG.rst (hunk 1 above)
Comment (nicoddemus) [inline CHANGELOG.rst]: Should have reviewed it first, sorry

File: CHANGELOG.rst (hunk 2 above)
Comment (nicoddemus) [inline CHANGELOG.rst]: Yes, result of a bad merge, my bad.

**Comment by nicoddemus** [comment]
Applied your suggestions directly on `features` in 02dd7d6. :grin:

**Pull request merged**